### PR TITLE
Black vegetable new hails

### PR DESCRIPTION
**Pull request by BlackVegetable**
Added some additional hails.

### CODE REVIEW

**Comment by KiLEdEnNis** [comment]
as far as I saw seems fine, now see what others think

File: data/hails.txt [hunk 1]
```diff
@@ -889,6 +911,7 @@ phrase "friendly civilian"
 		"are building a superweapon to destroy us all!"
 		"are secretly being controlled by the Lizard Illuminati!"
 		"are under the control of a hidden race of chipmunk aliens!"
+		"get together to play foosball every Thursday on Luna!"
```
Comment (Bladewood) [inline data/hails.txt]: https://en.wikipedia.org/wiki/Table_football in case anyone is confused

File: data/hails.txt [hunk 2]
```diff
@@ -974,6 +997,7 @@ phrase "friendly civilian"
 		" "
 	word
 		"trade routes, but I'm not"
+		"trade routes, but there is no way I'm"
```
Comment (Bladewood) [inline data/hails.txt]: Could we change "there is" to "there's"? It's correct, but it sounds a bit awkward to me.

File: data/hails.txt [hunk 3]
```diff
@@ -1355,6 +1379,7 @@ phrase "friendly syndicate"
 		"user-friendly products"
 		"ethics"
 		"offers"
+		"dental plans"
```
Comment (Bladewood) [inline data/hails.txt]: Would the Syndicate really offer this stuff?

File: data/hails.txt (hunk 1 above)
Comment (jafdy) [inline data/hails.txt]: foosball?

File: data/hails.txt (hunk 3 above)
Comment (MCOfficer) [inline data/hails.txt]: it's a common & successful way of making money, so i think they would.

File: data/hails.txt (hunk 2 above)
Comment (BlackVegetable) [inline data/hails.txt]: Sure thing.

File: data/hails.txt [hunk 4]
```diff
@@ -2721,6 +2748,7 @@ phrase "hostile Syndicate"
 		"Thank you for volunteering to be a target ship for Syndicated Weapon Systems!"
 		"Syndicated Systems disapproves of space littering; your ship's wreckage will be recycled after we are finished destroying it."
 		"With authority vested in me by Syndicated Systems corporate charter, section 8, paragraph 22... prepare to die!"
+                "The remains of your soon to be derelict ship will pad my profit margins!"
```
Comment (BlackVegetable) [inline data/hails.txt]: Hmm, I have an extra space here. Will fix.

**Comment by BlackVegetable** [comment]
There's some weird formatting around the Syndicate hostile hails. Is there a tab living there or something? I used only spaces for my new line and it doesn't quite match up. Also, will that break any code reading from the file?

File: data/hails.txt (hunk 4 above)
Comment (BlackVegetable) [inline data/hails.txt]: Actually, the other hails have something funny going on here. Will comment.

File: data/hails.txt (hunk 1 above)
Comment (BlackVegetable) [inline data/hails.txt]: Quite intentional. :)

File: data/hails.txt (hunk 4 above)
Comment (MCOfficer) [inline data/hails.txt]: where exactly? in front of "profit"? o.o

Comment (BlackVegetable) [inline data/hails.txt]: Enter the github online "edit" mode for this file. Select the whitespace behind any of the other Syndicate hails, and you'll see a different color for one character, which I think is a different whitespace character. Then do the same selection behind my new one. You'll see it doesn't have the same additional (different?) character. This is causing them to not line up in some of github's views.

**Comment by MCOfficer** [comment]
auswahl_004
loads of whitespaces, if that's what you mean...
(the ">>"s are hard tabs, the dots are whitespaces)

**Comment by BlackVegetable** [comment]
Huh, I should've just opened it with vim or something but I've been lazy here. I come from a python background where tabs are never used. Is this the only set of messages using tabs?

**Comment by Lineth** [comment]
I think it's always tabs in the coding here.

**Comment by MCOfficer** [comment]
as far as i can see, the entire file seems to be indented like this. only thing that stands out are these whitespaces.

**Comment by BlackVegetable** [comment]
I will purge the spaces to conform.

**Comment by BlackVegetable** [comment]
Fixed. Would you like this rebased/squashed into a single commit?

**Comment by BlackVegetable** [comment]
w.r.t. python, you are correct but I've never worked in a tab-codebase there and PEP8 recommends avoiding tabs: https://www.python.org/dev/peps/pep-0008/#id18

**Comment by KiLEdEnNis** [comment]
> Would you like this rebased/squashed into a single commit?

we can just click the "Files changed" button, so not needed

**Comment by Wrzlprnft** [comment]
while fewer commits are better, i'd add it to this pr.  then, when @endless-sky is back from vacation, you may need to edit it again, and then we'll see if we need to rebase it.

**Comment by BlackVegetable** [comment]
I already added the commit.

File: data/hails.txt (hunk 3 above)
Comment (Pointedstick) [inline data/hails.txt]: I think it's pretty funny!

File: data/hails.txt [hunk 5]
```diff
@@ -310,6 +316,7 @@ phrase "friendly civilian"
 		"everyone's"
 		"my crew is"
 		"some of my friends are"
+		"the older generation is"
```
Comment (ph2000bis) [inline data/hails.txt]: "I can't believe the older generation is following that new group"

Somehow sounds odd

Comment (BlackVegetable) [inline data/hails.txt]: Yeah, there's gotta be a better way to say that.

Comment (Pointedstick) [inline data/hails.txt]: "all the oldsters are" ?

Comment (Bladewood) [inline data/hails.txt]: Try "old-timer"?

File: data/hails.txt [hunk 6]
```diff
@@ -2721,6 +2748,7 @@ phrase "hostile Syndicate"
 		"Thank you for volunteering to be a target ship for Syndicated Weapon Systems!"
 		"Syndicated Systems disapproves of space littering; your ship's wreckage will be recycled after we are finished destroying it."
 		"With authority vested in me by Syndicated Systems corporate charter, section 8, paragraph 22... prepare to die!"
+		"The remains of your soon to be derelict ship will pad my profit margins!"
```
Comment (tehhowch) [inline data/hails.txt]: soon-to-be

File: data/hails.txt [hunk 7]
```diff
@@ -662,6 +678,7 @@ phrase "friendly civilian"
 		"Finally dropped off some"
 		"Dropped off my cargo of"
 		"Just made a big delivery of"
+		"Recently got rid of"
```
Comment (endless-sky) [inline data/hails.txt]: I don't think this phrase really works in the sentence.

Comment (BlackVegetable) [inline data/hails.txt]: Can you elaborate?

Comment (jafdy) [inline data/hails.txt]: `Recently got rid of garlic.` 
compared to
`Just delivered a load of garlic.`

Comment (BlackVegetable) [inline data/hails.txt]: I mean, both are equally sentence fragments. I can remove this one if you'd like though.

Comment (jafdy) [inline data/hails.txt]: Have you tried saying these out loud? The first one feels like the verbal equivalent of running into a brick wall. OK the second one is not "correct" but at the very least it doesn't feel awkward.

Comment (BlackVegetable) [inline data/hails.txt]: Hahaha. Saying them out loud made me understand. Hahahaha. Will remove.

**Comment by Pointedstick** [comment]
@BlackVegetable Can you make the last few requested changes so we can get this merged?

**Comment by BlackVegetable** [comment]
Yeah, I can make any adjustments in about 24 hours. Hopefully that'll work for y'all.

**Comment by BlackVegetable** [comment]
Ok, changes made. This should probably be rebased into a single commit, but I don't know your preferred git workflow fully, so I'll leave that up to you.

**Pull request merged**